### ISSUE TITLE
Add user login

### ISSUE THREAD

**Issue by cwkingjr** [charwking/cwking-api#51]
A verified user should be able to login with their email and password.

Successful login should respond with a secure httpOnly cookie containing an encrypted JWT access token, and possibly a non-secure cookie containing limited, safe (non-PII) data to allow client JS to access it for display purposes only (e.g., user name for "Logged in as: xxx").

**Comment by charwking** [comment]
I agree with the above, but I would say this is two separate things. First is the secure httpOnly cookie. And then later, as a separate task, we can add the non-secure cookie if needed.
